layouts/index.html

```diff
@@ -134,27 +134,23 @@ <h2 class="font-medium">Getting started</h2>
           <div class="bg-pattern-purple rounded p-6">
             <div class="flex flex-col gap-12">
               <div class="flex flex-col gap-4">
-                <h2 class="font-medium">Featured</h2>
+                <h2 class="font-medium">Research</h2>
                 <p class="text-xl">
-                  Accelerate your builds with cloud capabilities.
+                  Docker State of Application Development survey.
```

Comment:
Nit: The period here caught my eye. Probably correct as-is, but it seems more like a title in its current form. Suggest remove the period or make it more like a sentence.

Reply:
+1 to remove the period. We should probably remove the period from the Get started tile too. 

CC @dvdksn

Reply:
done